training/distributed_training/pytorch/model_parallel/gpt-j/train_gptj_smp_notebook.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>Optionally add a Conclusion section summarizing what the user accomplished in this notebook. Delete any resources that would incur cost, but I believe there's none to delete here as no model was deployed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>